src/components/jump-to-banner/jump-to-banner.js

```diff
@@ -0,0 +1,29 @@
+const jumpToBanner = () => {
+	if (document.querySelector(".js-jump-to")) {
+		const jumpToBanners = document.querySelectorAll(".js-jump-to");
+
+		jumpToBanners.forEach((jumpToBanner)=> {
+			let jumpToLinks = jumpToBanner.querySelectorAll(".js-jump-to__link");
+	
+			jumpToLinks.forEach((jumpToLink) => {
+				jumpToLink.addEventListener("click", function (e) {
+					e.preventDefault();
+					handleLinkClick(jumpToLink);
+				});
+			});
+
+			const handleLinkClick = (jumpToLink) => {
+				console.log("link clicked", jumpToLink);
+				let urlFromLink = new URL(jumpToLink.href);
+				let idFromURL = urlFromLink.hash.substring(1);
+				let sectionToScrollTo = document.getElementById(idFromURL);
+				window.scrollTo({
```

Comment:
you might want to consider https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView which I think would respect CSS scroll-margin